spec/rubocop/cop/style/hash_syntax_spec.rb

```diff
@@ -406,6 +428,17 @@
         expect_no_offenses('{ :a => :b, :c => :d }')
       end
     end
+
+    it 'registers an offense for implicit `call` method' do
+      expect_offense(<<~RUBY)
+        method(:puts).call(a: 0)
+                           ^^ Use hash rockets syntax.
+      RUBY
+
+      expect_correction(<<~RUBY)
+        method(:puts).call(:a => 0)
+      RUBY
```

Comment:
Shouldn't the code look like the following if it's "implicit"?

```suggestion
      expect_offense(<<~RUBY)
        method(:puts).(a: 0)
                           ^^ Use hash rockets syntax.
      RUBY

      expect_correction(<<~RUBY)
        method(:puts).(:a => 0)
      RUBY
```

Reply:
Yes, you're completely right. I don't know from where this `call` came from; I meant to use implicit variant everywhere. Fixed.